sagenb/flask_version/worksheet.py

```diff
@@ -350,7 +349,6 @@ def worksheet_delete_cell(worksheet):
     if len(worksheet.compute_cell_id_list()) <= 1:
         r['command'] = 'ignore'
     else:
-        prev_id = worksheet.delete_cell_with_id(id)
```

Comment:
Really, just remove this line? This is very wrong, as far as I can see, would lead to a runtime error.

Reply:
oops, no, it's fine - indeed, it's just a leftover from some older version, as far as I can see.